ui/client/components/graph/NodeDetailsContent.js

```diff
@@ -135,6 +135,10 @@ export class NodeDetailsContent extends React.Component {
     });
   };
 
+  validate(fieldValue, isRequired) {
```

Comment:
It does not validate "in general" it merely checks if the value is not empty.

Reply:
Done